lib/assessment.rb

```diff
@@ -20,20 +27,44 @@
 # Name and location should be writeable, but age should be read-only.
 
 ##
-# your code here
+class Person
+
+  attr_writer :name, :location
```

Comment:
I missed this earlier, but you should be using `attr_accessor` there, not `attr_writer`. `attr_writer` only creates a 'setter' method, _not a 'getter' one_. Since it's really rare that you'd want write access to a variable but _not_ read access, `attr_writer` is used extremely rarely.